blackduck/HubRestApi.py

```diff
@@ -509,6 +509,12 @@ def get_vulnerable_bom_components(self, version_obj, limit=9999):
             logging.warning("Failed to retrieve vulnerable bom components for project {}, status code {}".format(
                 version_obj, response.status_code))
 
+    def get_Component_Remediation(self, bom_component):
+        url = "{}/remediating".format(bom_component['componentVersion'])
+        print(url)
```

Comment:
instead of printing, which by definition goes to stdout, could you use logging instead, e.g.

logging.debug("url for getting remediation info is: {}".format(url))